docs/_includes/Statements.md

```diff
@@ -644,6 +640,42 @@ An `Assume` statement cannot be compiled. In fact, the compiler
 will complain if it finds an **assume** anywhere where it has not
 been replaced through a refinement steop.
 
+Using `...` as the argument of the statement is part of module refinement, as described [here](#sec-module-refinement).
+
+## Expect Statement
+
+````
+ExpectStmt =
+    "expect" { Attribute }
+    ( Expression(allowLemma: false, allowLambda: true)
+    | "..."
+    )
+    [ "," Expression(allowLemma: false, allowLambda: true) ]
+    ";"
+````
+
+The `expect` statement states a boolean expression that is checked to be true
+at run-time. That is, the compiler inserts into the run-time executable a 
+check that the given expression is true; if the expression is false, then
+the execution of the program halts immediately. If a second argument is
+given, it is a string that is included in the message emitted by the program
+when it halts; otherwise a default message is emitted.
+
+Because the expect expression is compiled, it cannot be a ghost expression.
+
+The `expect` statement is analogous to the `assert` statement. The `assert` expression
+is checked by the verifier and ignored at runtime; an `expect` expression is checked
+at runtime and ignored by the verifier.  Paired `assert` and `expect` statements checking the
```

Comment:
No, `expect` is not ignored by the verifier. To the verifier, an `expect` is like an `assume`.

Reply:
Corrected